lib/Stripe.php

```diff
@@ -71,4 +74,20 @@ public static function setVerifySslCerts($verify)
     {
         self::$verifySslCerts = $verify;
     }
+
+    /**
+     * @return string | null The stripe_account id for requests.
```

Comment:
And actually, mind just changing these two comments to read "The Stripe account ID for managed account requests." (Mostly just changing the casing on "stripe_account".)